File: Javascript/2-locating-elements/4-By-Class.js
const { Builder, By} = require('selenium-webdriver');

(async function(){
    const driver = new Builder().forBrowser('chrome').build();
    try{
        await driver.get('https://www.selenium.dev/')
        /**
         * Fetch the below DOM node
         * <div class="download-button-container">
         *   <a href="/downloads">
         *     <div class="download-button webdriver"><b>DOWNLOAD</b></div>
         *   </a>
         * </div>
         */
        let my_element = await driver.findElement(By.className("download-button-container"));
        console.log(await my_element.getText());
    } finally {
        await driver.quit()
    }

})();

File: Python/2-locating-elements/4-By-Class.py
from selenium import webdriver

driver = webdriver.Chrome()

driver.get('https://www.selenium.dev/')

'''
Fetch the below DOM node
<div class="download-button-container">
    <a href="/downloads">
    <div class="download-button webdriver"><b>DOWNLOAD</b></div>
    </a>
</div>
'''
my_class_element = driver.find_element_by_class_name('download-button-container')

# Print the innerText of the DOM node fetched
print(my_class_element.text)

driver.quit()